model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -82,6 +82,27 @@ def replace_pattern(self, graph: Graph, match: dict):
             log.debug('The value of the "maximum_y_data" is not defined or is not constant')
             return
 
+        # We need to check axes which performed reduction because IE supports only 2D, 3D, 4D inputs and
+        # reduction only along spatial dimensions.
+        input_shape = match['sum'].in_port(0).data.get_shape().size
```

Comment:
please rename this variable to `input_rank`. I got confusion.

Reply:
Done.